staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/start.go

```diff
@@ -90,6 +90,13 @@ func (o CustomResourceDefinitionsServerOptions) Validate(args []string) error {
 	return utilerrors.NewAggregate(errors)
 }
 
+func (o *CustomResourceDefinitionsServerOptions) IsRegistered(args []string) error {
+	errors := []error{}
+	errors = append(errors, o.RecommendedOptions.Validate()...)
```

Comment:
Validate in IsRegistered?

Reply:
I think this is a mistake, I will double check.

Reply:
Yes, this is useless, I must delete it.